sources/packages/backend/apps/api/src/route-controllers/designation-agreement/models/designation-agreement.dto.ts

```diff
@@ -75,6 +75,11 @@ export class SubmitDesignationAgreementAPIInDTO {
   @ValidateNested({ each: true })
   @Type(() => SubmittedLocationsAPIInDTO)
   locations: SubmittedLocationsAPIInDTO[];
+
```

Comment:
Please remove the extra blank line to be aligned with the other property in the same DTO.